src/utils.rs

```diff
@@ -670,6 +672,8 @@ pub struct Preferences {
 	pub hide_score: String,
 	#[revision(start = 1)]
 	pub remove_default_feeds: String,
+	#[revision(start = 1)]
+	pub clean_urls: String,
```

Comment:
Please review `revision` [docs](https://docs.rs/revision/latest/revision/) to handle this - specifically, you'll want to increment the upper level struct's revision to 2, set the start for `clean_urls` to 2, and read the docs on how you'll handle reverse compatibility (default to off).